main.tf

```diff
@@ -14,6 +14,18 @@ resource "aws_cloudfront_distribution" "api_gate" {
     }
   }
 
+  origin {
+    domain_name = var.mdw_fqdn
+    origin_id   = "api_mdw"
+
+    custom_origin_config {
+      http_port              = 80
+      https_port             = 443
+      origin_protocol_policy = "http-only"
```

Comment:
This is external service so https is required, https-only ?

Reply:
0c1083a